### PR TITLE
Adds more dresses and clothings

### PR DESCRIPTION
**Pull request by totallynotkatfood**
## About The Pull Request

adds more dresses and clothings, also the accursed wedding crate and kimonos

## Why It's Good For The Game

more stuffs, this is probably gonna be closed anyways

## Changelog
:cl:
add: Added more clothes to vendors and aushop
add: Added the wedding crate
imageadd: added some icons for clothings
/:cl:

### CODE REVIEW

**Comment by jbox144** [comment]
Now all we need is a bride headpiece and the ability for borgs to wear it.

**Comment by MCterra10** [comment]
"this is probably gonna be closed anyway"
Guilt tripping the host smh smh smh

**Comment by totallynotkatfood** [comment]
ok terra

File: austation/code/modules/vending/clothesmate.dm [hunk 1]
```diff
@@ -1,4 +1,6 @@
-/obj/machinery/vending/autodrobe
+/obj/machinery/vending/clothing
```
Comment (TheFakeElon) [inline austation/code/modules/vending/clothesmate.dm:1]: ?

**Review by TheFakeElon — APPROVED** [review]

**Pull request merged**